src/components/filmPage/filmInfo/filmInfo.module.scss

```diff
@@ -0,0 +1,161 @@
+.mainContainer {
+  display: flex;
+  width: 100%;
+
+  box-sizing: border-box;
+  overflow: hidden;
+  margin-top: -6vh;
+  position: relative;
+  padding: 0 20vh 20vh;
+
+  .filmPosterContainer {
+    // styles for skeleton
+    .loadingImage {
+      width: 15vw;
+      height: 41vh;
+      border-radius: 15px;
+      background-color: #3f3f3f;
+
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      svg rect {
+        fill: var(--golden);
+      }
+    }
+    //styles for skeleton
```

Comment:
This one duplicates comment above. Remove both or rename one of them

Reply:
fixed